go/test/endtoend/vtgate/queries/aggregation/distinct_test.go

```diff
@@ -29,8 +29,8 @@ func TestDistinct(t *testing.T) {
 	mcmp.Exec("insert into t7_xxhash(uid,phone) values('1',4), ('2',4), ('3',3), ('4',1), ('5',1)")
 	mcmp.Exec("insert into aggr_test(id, val1, val2) values(1,'a',1), (2,'A',1), (3,'b',1), (4,'c',3), (5,'c',4)")
 	mcmp.Exec("insert into aggr_test(id, val1, val2) values(6,'d',null), (7,'e',null), (8,'E',1)")
-	mcmp.AssertMatches("select distinct val2, count(*) from aggr_test group by val2", `[[NULL INT64(2)] [INT64(1) INT64(4)] [INT64(3) INT64(1)] [INT64(4) INT64(1)]]`)
-	mcmp.AssertMatches("select distinct id6 from t3 join t7_xxhash on t3.id5 = t7_xxhash.phone", `[[INT64(3)] [INT64(5)]]`)
+	mcmp.Exec("select distinct val2, count(*) from aggr_test group by val2")
+	mcmp.Exec("select distinct id6 from t3 join t7_xxhash on t3.id5 = t7_xxhash.phone")
```

Comment:
any reason for changing in this file? I do not see using the block join here.

Reply:
We care about having the same results as MySQL, no need to spell out the exact expectation when we want the same as MySQL.